src/components/purchase-complete/index.js

```diff
@@ -64,38 +64,44 @@ const PurchaseComplete = ({
 
     let orderCompleteButtonText = (
         currentUserTicket && requireExtraQuestions ?
-            rest.hasOwnProperty('initialOrderCompleteButton') && typeof rest.initialOrderCompleteButton !== 'undefined' ?
+            rest.hasOwnProperty('initialOrderCompleteButton') && typeof rest.initialOrderCompleteButton === 'string' 
+            && rest.initialOrderCompleteButton.trim().length > 0 ?
                 rest.initialOrderCompleteButton
                 :
                 T.translate('purchase_complete_step.initial_order_complete_button')
             :
-            rest.hasOwnProperty('orderCompleteButton') && typeof rest.orderCompleteButton !== 'undefined' ?
+            rest.hasOwnProperty('orderCompleteButton') && typeof rest.orderCompleteButton === 'string' 
```

Comment:
could we change this to an utility method
like
function isEmptyString(val){
return typeof val === 'string' 
            && val.trim().length > 0;
}
??